### PR TITLE
ci(release): use gh workflow to manually release and push

### PR DESCRIPTION
**Pull request by markov00**
## Summary

With the idea of dismiss Travis CI for release, I'm adding a new gh workflow that can be manually triggered and do both release and gh pages deploy.

I'm using this https://github.com/marketplace/actions/github-pages-action for deploy.
Few security tokens need to be added for `semantic-release` before merging on master

### CODE REVIEW

File: .github/workflows/auto_release.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Testing
+        run: yarn test --ci
+      - name: Semantic-Release
+        run: yarn semantic-release
+      - name: Build static storybook
+        run: yarn storybook:build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (nickofthyme) [inline .github/workflows/auto_release.yaml:36]: These secrets are built into github?

**Review by nickofthyme — APPROVED** [review]
LGTM 🎉

File: .github/workflows/auto_release.yaml (hunk 1 above)
Comment (markov00) [reply .github/workflows/auto_release.yaml:36]: it should be https://github.com/marketplace/actions/github-pages-action#%EF%B8%8F-set-runners-access-token-github_token
> GitHub Actions runner automatically creates a GITHUB_TOKEN secret to use in your workflow. You can use the GITHUB_TOKEN to authenticate in a workflow run.

**Pull request merged**